slack-api-client/src/main/java/com/slack/api/methods/impl/TeamIdCache.java

```diff
@@ -32,15 +32,16 @@ public String lookupOrResolve(String token) {
                 FormBody.Builder form = toForm(AuthTestRequest.builder().token(newToken).build());
                 Response response = methodsImpl.runPostFormWithToken(form, Methods.AUTH_TEST, token);
                 AuthTestResponse authTest = methodsImpl.parseJsonResponseAndRunListeners(null, null, response, AuthTestResponse.class);
-                if (authTest.isOk()) {
+                if (authTest != null && authTest.isOk()) {
```

Comment:
While writing a mock API server and doing tests with it, I found the potential null pattern. Except the situation where the Slack server-side returns empty response body, `authTest` cannot be null, though.